tools/downloader/README.md

```diff
@@ -435,6 +435,9 @@ describing a single model. Each such object has the following keys:
 
   Additional possible values might be added in the future.
 
+* `quantization_output_precisions`: the list of precisions that the model can be quantized to by
+  the model quantizer. Possible values are the same as for `precisions`.
```

Comment:
This is not quite true. Right now possible values are: FP32-INT8, FP16-INT8

Reply:
Yeah, fair point. Fixed.